### ISSUE TITLE
tridentctl: better kubectl output parsing

### ISSUE THREAD

**Issue by george-angel** [NetApp/trident#670]
Trident: 21.10.0

```
$ /tmp/trident-installer/./tridentctl -n sys-trident version
Error: invalid character 'I' looking for beginning of value
```

Probably want to parse stderr separately:

```
$ kubectl --context exp-1-merit -n sys-trident get pod
I1104 09:12:07.428631  234537 request.go:665] Waited for 1.084555807s due to client-side throttling, not priority and fairness, request: GET:https://lb.master.exp-1.merit.uw.systems/apis/cert-manager.io/v1beta1?timeout=32s
NAME                           READY   STATUS    RESTARTS   AGE
trident-csi-2ccp7              2/2     Running   0          26d
trident-csi-556b9cb7cb-9d8hc   6/6     Running   0          8d
trident-csi-5xqts              2/2     Running   0          13d
```

```
$ kubectl --context exp-1-merit -n sys-trident get pod 2>/dev/null
NAME                           READY   STATUS    RESTARTS   AGE
trident-csi-2ccp7              2/2     Running   0          26d
trident-csi-556b9cb7cb-9d8hc   6/6     Running   0          8d
trident-csi-5xqts              2/2     Running   0          13d
```

**Comment by oleimann** [comment]
It's probably easier to recognize such I/W/E#### messages in the parser and drop them, then it is to change the I/O channel reading, or in java terms: if(!line.matches("^[IWE]\d+") { parse; }

**Comment by gnarl** [comment]
This issue is fixed in [commit bc247fb](https://github.com/NetApp-openstack-dev/trident/commit/bc247fbfa64e47ca27590aee5e1bda749882be9b) and will be included in the Trident 22.07 release.
